cmd/root.go

```diff
@@ -8,7 +8,21 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/Pallinder/go-randomdata"
+	"github.com/segmentio/analytics-go"
 	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/pkg/analyticsutil"
+)
+
+const (
+	analyticsEndpoint = "https://analytics.starport.cloud"
```

Comment:
let's make sure we have lower response times before merging

ccc @helder-moreira

Reply:
thanks, it's pretty fast now :)